api/circulation.py

```diff
@@ -78,7 +95,16 @@ def __repr__(self):
 
 class HoldInfo(CirculationInfo):
 
-    """A record of a hold."""
+    """A record of a hold.
+
+    :param identifier_type Ex.: Identifier.ONECLICK_ID.
+    :param identifier Expected to be the unicode string of the isbn, etc..
+    :param start_date When the patron made the reservation.
+    :param end_date When reserved book is expected to become available.  Expected to be passed in 
+        date, not unicode format.
+    :param hold_position  Patron's place in the hold line.  
+        When not available, default to be passed is 0.
```

Comment:
When hold_position is not available, the value passed in should be None, not 0. hold_position zero means that the patron is first in line to borrow the book and can check it out immediately.